.github/workflows/test-e2e.yml

```diff
@@ -205,6 +205,7 @@ jobs:
           # one job may wait for deploys in other jobs (only one deploy may be in progress for
           # a given alias at a time), resulting in cascading timeouts.
           DEPLOY_ALIAS: vercel-next-e2e-${{ matrix.version_spec.selector }}-${{ matrix.group }}
+          NODE_OPTIONS: --import /home/runner/work/opennextjs-netlify/opennextjs-netlify/opennextjs-netlify/skip-integrations.mjs
```

Comment:
there are sometimes failure related to fetching extensions, this might be due to doing a lot of deploys for the same site/account and hitting rate limiting or something, it's not quite clear - this for now just mocks responses to be empty to avoid those failures and see if that helps at all